src/pages/home/Home.scss

```diff
@@ -1,157 +1,189 @@
+          border-bottom: none;
         }
-  
-        &__search {
-          th {
-            padding-top: 10px;
-            padding-bottom: 0px;
-            border-bottom: none;
-          }
 
-          .search {
-            border: 1px solid var(--border);
-            border-radius: 5px;
-            min-height: 40px;
-            width: 85%;
-            margin-top: 1%;
-            margin-left: 1%;
-            margin-bottom: 2%;
-  
-            svg {
-              margin: 0 5px 0 5px;
-            }
-          }
-  
-          &__filter {
-            display: flex;
-            justify-content: flex-start;
-            align-items: center;
-  
-            label {
-              padding-right: 10px;
-            }
+        .search {
+          border: 1px solid var(--border);
+          border-radius: 5px;
+          min-height: 40px;
+          width: 85%;
+          margin-top: 1%;
+          margin-left: 1%;
+          margin-bottom: 2%;
+
+          svg {
+            margin: 0 5px 0 5px;
           }
+        }
+
+
+        &__filter {
+          display: flex;
+          justify-content: flex-start;
+          align-items: center;
 
-          .advanced__search__button {
-            margin-bottom: 1%;
-            margin-left: 1%;
-            text-transform: none;
-            color:grey;
+          label {
+            padding-right: 10px;
           }
         }
-  
-        &__tr {
-          @media (min-width: 768px) {
-            display: table-row;
-  
-            td {
-              &:first-child {
-                padding-left: 30px;
-              }
-  
-              &:last-child {
-                padding-right: 30px;
-              }
+
+        .advanced__search__button {
+          margin-bottom: 1%;
+          margin-left: 1%;
+          text-transform: none;
+          color: grey;
+        }
+      }
+
+
+      &__tr {
+        @media (min-width: 768px) {
+          display: table-row;
+
+          td {
+            &:first-child {
+              padding-left: 30px;
+            }
+
+            &:last-child {
+              padding-right: 30px;
             }
-          }
-  
-          th {
-            padding-bottom: 20px;
-            font-weight: 400;
-            color: var(--text-faded);
-            text-align: left;
           }
         }
-        tbody tr {
-          height: 50px;
+
+        th {
+          padding-bottom: 20px;
+          font-weight: 400;
+          color: var(--text-faded);
+          text-align: left;
         }
       }
-    }
 
-    .search__buttons {
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-        margin-bottom: 24px;
-        margin-left: 35%;
-        margin-right: 5%;
+      tbody tr {
+        height: 50px;
       }
+    }
   }
 
+
+  .search__buttons {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    margin-left: 5%;
```

Comment:
in main, the margin-left is set at 35% so make sure to change it or else the radio buttons appear too much to the left.

Reply:
Done, thanks for noticing